src/backend/gporca/libgpopt/src/operators/CLogicalRowTrigger.cpp

```diff
@@ -33,7 +33,8 @@ CLogicalRowTrigger::CLogicalRowTrigger(CMemoryPool *mp)
 	  m_rel_mdid(NULL),
 	  m_type(0),
 	  m_pdrgpcrOld(NULL),
-	  m_pdrgpcrNew(NULL)
+	  m_pdrgpcrNew(NULL),
+	  m_efs(IMDFunction::EfsImmutable)
```

Comment:
The constructor you mentioned is used only in CXformFactory::Instantiate -> CXformImplementRowTrigger() -> CLogicalRowTrigger(). We can't use EfsSentinel as it's nor a real value, but just a handy representation of max value (useful to allocate arrays, use in loops, etc). As for other values, it seems we can use any of them, but I use the same value as in second constructor to keep uniformity. Any value of m_efs should not be result of error here (even if it's a 2139062143)

Reply:
> We use immutable because the possibility of promotion to volatile is hard-wired into the code, and not vice versa.

It seems we are not talking about the second constructor here